README.md

```diff
@@ -1,28 +1,35 @@
-# Gruppe Adler Mod
+# Gruppe Adler Extras
 
-[![GitHub tag](https://img.shields.io/github/tag/gruppe-adler/gruppe_adler_mod.svg)](https://github.com/gruppe-adler/gruppe_adler_mod/releases)
-![build](https://github.com/gruppe-adler/gruppe_adler_mod/workflows/CI/badge.svg)
-![ACE SQF validator](https://github.com/gruppe-adler/gruppe_adler_mod/workflows/ACE%20SQF%20validator/badge.svg)
+[![GitHub tag](https://img.shields.io/github/tag/gruppe-adler/grad_extras.svg)](https://github.com/gruppe-adler/grad_extras/releases)
+![build](https://github.com/gruppe-adler/grad_extras/workflows/CI/badge.svg)
 
-Gruppe Adler Mod is a collection of Gruppe Adler original addons that add new functionality or modify and improve existing functionality to suit our Arma gameplay.
+Gruppe Adler Extras is a collection of Gruppe Adler original addons that add new functionality or modify and improve existing functionality to suit our Arma gameplay. The main difference to [Gruppe Adler Mod](https://github.com/gruppe-adler/grad_mod) is that addons included in this mod can have additional dependencies (other than CBA / ACE).
 
 ## Dependencies
-TFAR
-RHS
-ACE3
-CBA
+- CBA
+- ACE3
+- TFAR
+- RHS
+- GM
+- NIArms G3 / MG3
 
 ## For Developers
 
-We're using [HEMTT](https://github.com/synixebrett/HEMTT)
+We're using [HEMTT](https://github.com/synixebrett/HEMTT).
 
 ### Releasing
-
-* update `addons\main\script_version.hpp`
-* do use `hemtt build --release`
-* create tag on GitHub
-* upload your build
+- Update `addons\main\script_version.hpp`
+- Commit your changes
+- Create tag on GitHub
+- NO NEED to upload the built mod. A GitHub Action workflow will take care of that
 
 ### Building
+On Linux:
+```
+hemtt build --release
+```
+On Windows:
+```
+hemtt.exe build --release
```

Comment:
Needs P Drive and the following script to resolve some issues with hemtt:
```suggestion
start build-hemtt.bat 
```

start build-hemtt.bat